Cargo.lock

```diff
@@ -3007,9 +3007,9 @@ dependencies = [
 
 [[package]]
 name = "proc-macro2"
-version = "1.0.71"
+version = "1.0.73"
```

Comment:
@Turbo87 you might want to roll this back and pin proc-macro2 to 1.0.71 until the IDE is fixed (https://youtrack.jetbrains.com/issue/RUST-13133) because 1.0.72 has a mechanism that is similar to what was causing https://github.com/dtolnay/thiserror/issues/270 for you.

Reply:
interesting... I've been developing on top of this PR today and haven't noticed any test recompilations. I wonder what is different to the other scenario that triggers the issue 🤔

Reply:
I just ran `cargo clean` and after that unfortunately the issue started to appear. 

I'm not sure why `cargo clean` has suddenly triggered it though. It looks like cargo may have cached something incorrectly?

I've disabled the "Show test results in the Test output window" option in my IDE (as described in https://youtrack.jetbrains.com/issue/RUST-13133) and that helped for now, but it's not a good long-term solution. Let's hope the JetBrains people implement the proposed fix soon :)